src/environment_manager.py

```diff
@@ -15,7 +15,6 @@
 from src import util
```

Comment:
New PODs shouldn't modify (or need to modify) framework code in `/src`. Please commit the changes from `git checkout develop -- src` to bring the contents of `/src` back to what's in the develop branch.

Reply:
Done in afb8f0a